runsc/boot/loader.go

```diff
@@ -52,6 +52,7 @@ import (
 	"gvisor.dev/gvisor/pkg/sentry/seccheck"
 	pb "gvisor.dev/gvisor/pkg/sentry/seccheck/points/points_go_proto"
 	"gvisor.dev/gvisor/pkg/sentry/socket/netfilter"
+	externalstack "gvisor.dev/gvisor/pkg/sentry/stack"
```

Comment:
Nit: I'd probably import this as `sentrystack`.

Reply:
Good point. Thx for advice. We will address it in next version soon.